packages/jest-serializer/src/__tests__/index.test.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+import prettyFormat from 'pretty-format';
```

Comment:
Right! good catch!

Reply:
Oh, actually no 😅. There's no way you can develop `jest` without having `pretty-format` (`yarn` complained when trying to add it, and it ended up adding it to the root `package.json` instead of the child one).

Reply:
Ahh, I thought about this because it's going to be a standalone package, but then again, it's still inside a workspace monorepo. 👍